cmd/agent/app/options/options.go

```diff
@@ -182,6 +245,8 @@ func NewGrpcProxyAgentOptions() *GrpcProxyAgentOptions {
 		ProbeInterval:             1 * time.Second,
 		SyncIntervalCap:           10 * time.Second,
 		ServiceAccountTokenPath:   "",
+		ApiServerMapping:          portMapping{LocalPort: 6443, RemoteHost: "localhost", RemotePort: 6443},
```

Comment:
I would prefer no default values so this feature is off unless both the feature gate is on and these flags are set. Also 6443 seems an odd remote port for the KAS.

Reply:
@cheftako I did it this way to match the default secure port of KAS that is `6443`, note that the two have to match for this feature to work (see the [KEP](https://github.com/kubernetes/enhancements/tree/846705f347e51537fbef5835b321f5933d088b5c/keps/sig-cloud-provider/2025-extend-konnectivity-for-both-directions#handling-the-traffic-from-the-pods-to-the-agent)). I thought that the feature gate was enough, but if you prefer this to be specified to enable the feature I can do it.

Reply:
done